app/components/home-header.tsx

```diff
@@ -46,7 +50,9 @@ const HomeHeader = ({
         justifyContent="space-between"
       >
         <ReportAddress searchedAddress={searchedAddress} />
-        <Share />
+        <Suspense>
```

Comment:
Wrapping our code that has client-specific bits w/`<Suspense>` appears to be the correct for the prerender errors we were seeing on eg Next 14.